src/activities/guesscount-summer/guesscount-summer.js

```diff
@@ -0,0 +1,196 @@
+/* GCompris - guesscount-summer.js
+ *
+ * Copyright (C) 2016 YOUR NAME <xx@yy.org>
+ *
+ * Authors:
+ *   <THE GTK VERSION AUTHOR> (GTK+ version)
+ *   "YOUR NAME" <YOUR EMAIL> (Qt Quick port)
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+.pragma library
+.import QtQuick 2.0 as Quick
+.import GCompris 1.0 as GCompris
+
+var url = "qrc:/gcompris/src/activities/guesscount-summer/resource/"
+/*var operators = [
+            "+",
+            "-",
+            "/",
+            "*",
+        ]*/
+var default_operators
```

Comment:
I would directly assign them here, not in a file. Can the values change depending on the level? If no, you can just assign it to ["+","-","/","*"]